Automattic/wp-calypso
# Display notice when a plugin is active without a subscription

Related to #69973

## Proposed Changes

- active / inactive status displayed on the plugin browser item doesn't depend on having a subscription it only depends on the actual plugin status reported from the API.
- In the plugin details page, if there isn't an active subscription, a notice with the message is displayed "Plugin subscription not found or you have purchased the plugin outside of WordPress.com. Purchase a WordPress.com subscription if you want to receive updates and support"

## Testing Instructions

- Apply this patch
- Download Yoast SEO Premium directly from their site: https://yoast.com/wordpress/plugins/seo/
- Install it on AT WordPress.com site.
- Search for yoast in Marketplace /plugins.
- It will display Installed and Active.

<img width="1557" alt="Screenshot 2023-04-04 at 3 32 47 PM" src="https://user-images.githubusercontent.com/351784/230155612-671e70ae-cdbc-462a-972c-e17aed17aba2.png">

- Go to the details page of the plugin
- A notice is displayed at the top with the message

<img width="1573" alt="Screenshot 2023-04-05 at 12 25 24 PM" src="https://user-images.githubusercontent.com/351784/230157189-edbd0fdd-8854-4526-8ab1-fcdc7d261de7.png">

- This notice should not be displayed in preinstalled or plugins that don't belong to the marketplace.
- Upgrade a site from a simple to an e-commerce plan and make sure the notice doesn't appear on bundled plugins for instance WooCommerce Product Add-ons.

## Pre-merge Checklist
